File: Source_code_BUIDUCLONG/Java_OOP/src/mypack/NhaSanXuat.java
package mypack;

import java.util.List;

public class NhaSanXuat {
	 public String maNSX; // Mã nhà sản xuất
     public String tenNSX; // Tên nhà sản xuất
     public String quocGia; // Quốc gia
     
     public NhaSanXuat(String maNSX, String tenNSX, String quocGia, List<ThietBi> dsThietBi)
     {
         this.maNSX = maNSX;
         this.tenNSX = tenNSX;
         this.quocGia = quocGia;
         this.dsThietBi = dsThietBi;
     }
     public NhaSanXuat(NhaSanXuat nsx)
     {
         maNSX = nsx.maNSX;
         tenNSX = nsx.tenNSX;
         quocGia = nsx.quocGia;
         dsThietBi = nsx.dsThietBi;
     }
     public List<ThietBi> dsThietBi;
     
     public String getMaNSX()
     {
     	return maNSX;
     }
     public void setMaNSX (String maNSX)
     {
     	this.maNSX = maNSX;
     }
     public String getTenNSX()
     {
     	return tenNSX;
     }
     public void setTenNSX (String tenNSX)
     {
     	this.tenNSX = tenNSX;
     }
     public String getQuocGia()
     {
     	return quocGia;
     }
     public void setQuocGia(String quocGia)
     {
     	this.quocGia = quocGia;
     }
     public List<ThietBi> getDsPhieuNhap()
     {
     	return dsThietBi;
     }
     public void setDsPhieuNhap(List<ThietBi> dsThietBi)
     {
     	this.dsThietBi = dsThietBi;
     }
}

File: Source_code_BUIDUCLONG/C#_OOP/NhaSanXuat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CK_OOP_BuiDucLong
{
    internal class NhaSanXuat
    {
        public string maNSX { get; set; } // Mã nhà sản xuất
        public string tenNSX { get; set; } // Tên nhà sản xuất
        public string quocGia { get; set; } // Quốc gia
        
        public NhaSanXuat(string maNSX, string tenNSX, string quocGia, ICollection<ThietBi> dsThietBi)
        {
            this.maNSX = maNSX;
            this.tenNSX = tenNSX;
            this.quocGia = quocGia;
            this.dsThietBi = dsThietBi;
        }
        public NhaSanXuat(NhaSanXuat nsx)
        {
            maNSX = nsx.maNSX;
            tenNSX = nsx.tenNSX;
            quocGia = nsx.quocGia;
            dsThietBi = nsx.dsThietBi;
        }
        public ICollection<ThietBi> dsThietBi { get; set; }

    }
}
